runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ProsSection/ProsSection.module.scss

```diff
@@ -0,0 +1,82 @@
+@import '#src-landing/scss/main.scss';
+
+$MIN-CARD-WIDTH: 280px;
+$ONE-COLUMN-BREAKPOINT: 600px;
+
+.root {
+  composes: root from '#src-landing/components/SectionContent/SectionContent.module.scss';
+  display: grid;
+  height: max-content;
+  grid-gap: 27px 23px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-flow: dense;
+  grid-auto-rows: minmax(80px, auto);
+}
+
+.box {
+  font-size: 150%;
```

Comment:
150% of what? Imo there should be `em` or `rem` used instead, depends on the use case